scripts/scene/RoomBuilder.gd

```diff
@@ -32,13 +32,13 @@ var popup_message: String
 
 enum Action {BACK, FORWARD, COMPLETE}
 
-func _init(gui: GUI, station: Station, base_tile_map: TileMap, build_tile_map: TileMap, rooms: Array[Room], room_types: Array[RoomType]) -> void:
-	self.gui = gui
-	self.station = station
-	self.base_tile_map = base_tile_map
-	self.build_tile_map = build_tile_map
-	self.rooms = rooms
-	self.room_types = room_types
+func _init(gui_node: GUI, station_node: Station, base_tile_map_node: TileMap, build_tile_map_node: TileMap, rooms_arr: Array[Room], room_types_arr: Array[RoomType]) -> void:
```

Comment:
ugly naming i know but was generating a shadowing error

Reply:
No, it's a fair change! Thanks.